src/main/resources/lib/enonic/static/index.es6

```diff
@@ -3,9 +3,11 @@ const optionsParser = require('/lib/enonic/static/options');
 const ioLib = require('/lib/enonic/static/io');
 const runMode = require('/lib/enonic/static/runMode');
 
-const getResponse200 = (path, resource, contentTypeFunc, cacheControlFunc, etag) => {
-    const contentType = contentTypeFunc(path, resource);
-    const cacheControlHeader = cacheControlFunc(path, resource, contentType);
+const getResponse200 = (path, fallbackPath, resource, contentTypeFunc, cacheControlFunc, etag) => {
+    const contentType = contentTypeFunc(fallbackPath || path, resource);
+    const cacheControlHeader = fallbackPath
+        ? 'must-revalidate'
```

Comment:
```suggestion
        ? 'no-cache'
```
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#requiring_revalidation

Reply:
Fixed. Also, merged in docs and updated tests. Should be all good now.